extendr-api/src/metadata.rs

```diff
@@ -96,6 +96,14 @@ fn write_doc(w: &mut Vec<u8>, doc: &str) -> std::io::Result<()> {
     Ok(())
 }
 
+/// Wraps invalid R identifers, like `_function_name`, into backticks.
+fn sanitize_identifier(name: &str) -> String {
+    match name.chars().next() {
+        Some('_') => format!("`{}`", name),
+        _ => name.to_string(),
+    }
```

Comment:
Sorry for nitpicking comment..., using `starts_with()` looks easier to read to me (but this might be just my preference).

``` rust
    if name.starts_with('_') {
        format!("`{}`", name)
    } else {
        name.to_string()
    }
```